front/lib/api/data_source_view.ts

```diff
@@ -276,3 +282,58 @@ export async function getContentNodesForDataSourceView(
     total: contentNodesResult.total,
   });
 }
+
+type PatchDataSourceViewResult = Result<
+  { dataSourceView: ReturnType<DataSourceViewResource["toJSON"]> },
+  { status: number; message: string }
+>;
+
+export async function handlePatchDataSourceView(
+  req: NextApiRequest,
+  auth: Authenticator,
+  dataSourceView: DataSourceViewResource
+): Promise<PatchDataSourceViewResult> {
```

Comment:
Usually we keep the handlers with Request/Response inside pages handler, you can just move this function to `front/pages/api/w/[wId]/vaults/[vId]/data_source_views/[dsvId]/index.ts`

Reply:
Didn't really know where to put it tbh, will move to `front/pages/api/w/[wId]/vaults/[vId]/data_source_views/[dsvId]/index.ts`